api/envoy/config/filter/http/jwt_authn/v2alpha/config.proto

```diff
@@ -183,44 +183,84 @@ message JwtHeader {
   string value_prefix = 2;
 }
 
+// This message specifies a Jwt requirement.
+//
+// .. code-block:: yaml
+//
+//    required_issuers: [jwt_issuer1, jwt_issuer2]
+//
+// PerRouteConfig for jwt_authn filter uses this message.
+message JwtRequirement {
+  // Specify list of required Jwt issuers. If list is empty,  Jwt is NOT required.
+  // For multiple required_issuers, all are required.
+  repeated string required_issuers = 1;
+
+  // It only applies when required_issuers field is not empty.
+  // If true, the request is allowed if JWT is missing or JWT verification fails.
+  // Default is false, a request without JWT or failed JWT verification is not allowed.
+  bool allow_missing_or_failed = 2;
```

Comment:
It's not quite clear to me why we need this bool? Can we simply put empty required_issuers list?

Reply:
Sound likes a good idea.  I will remove it.